### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by tracob-github** [neel-dey/Atlas-GAN#1]
Hi, thanks for providing such an excellent work. Could you provide your preprocessing dataset?

**Comment by neel-dey** [comment]
Hi @tracob-github, thanks for your interest! Unfortunately, while the dHCP dataset used in the paper is public, it cannot be reshared person-to-person according to their terms of use (pt 9 [here](http://www.developingconnectome.org/data-release/second-data-release/open-access-dhcp-data-terms-of-use-version-4-0_2019-05-23/)). If you're interested, I would be happy to provide high-level pseudocode and/or subject indices for you to arrive at the same dataset.

As far as I know, Predict-HD is not public yet, but you could get in touch with Hans Johnson at the University of Iowa to ask about it.

**Comment by kvttt** [comment]
Hi @neel-dey, I am wondering if you could provide more details on how to arrive at the same dataset from the original dHCP dataset. Specifically, I would really appreciate if you could explain how to normalize the ages and the images (if any). Thank you in advance!

**Comment by kvttt** [comment]
> Hi @tracob-github, thanks for your interest! Unfortunately, while the dHCP dataset used in the paper is public, it cannot be reshared person-to-person according to their terms of use (pt 9 [here](http://www.developingconnectome.org/data-release/second-data-release/open-access-dhcp-data-terms-of-use-version-4-0_2019-05-23/)). If you're interested, I would be happy to provide high-level pseudocode and/or subject indices for you to arrive at the same dataset.
> 
> As far as I know, Predict-HD is not public yet, but you could get in touch with Hans Johnson at the University of Iowa to ask about it.


Hi @neel-dey, I am wondering if you could provide more details on how to arrive at the same dataset from the original dHCP dataset. I specifically would like to know how to normalize the ages and the images (if any). Thank you in advance for your help!

**Comment by neel-dey** [comment]
I didn't do anything special for normalization there, just divided the ages (in terms of gestational age in weeks) of all subjects by the maximum value in the training set (which was 45ish if I remember correctly) and divided the image intensities by the image-specific maximum value (after non-brain regions were masked out). 

W.r.t. the rest of the details on how dHCP release 2 was preprocessed for this paper, they're in Appendix B.1.2 [here](https://arxiv.org/pdf/2105.04349.pdf). I believe release 3 is out now and that's likely a great dataset to try this on.

If you need any other details, feel free to ask/open a new issue - happy to help :)
